packages/dashboard/src/managers/resource-manager-robots.ts

```diff
@@ -53,6 +54,28 @@ export class RobotResourceManager {
     }
   };
 
+  getRobotIconScale = (fleetName: string, robotModel?: string | undefined): number | null => {
+    if (!this.fleetExists(fleetName)) {
+      debug(`failed to load scale for "${fleetName}/${robotModel}" (fleet not in resources)`);
+      return null;
+    }
+
+    if (!this.robots[fleetName].hasOwnProperty('scale')) {
+      debug(
+        `failed to load scale for "${fleetName}/${robotModel}" (fleet/model does not have an scale)`,
+      );
+      return null;
+    }
+
+    const robotScale = this.robots[fleetName].scale;
+
+    if (!!robotModel && robotScale.hasOwnProperty(robotModel)) {
```

Comment:
```suggestion
    if (!!robotModel) {
```
We won't need to check for this `robotModel` property